src/core_ocean/shared/mpas_ocn_time_varying_forcing.F

```diff
@@ -0,0 +1,461 @@
+!>  over oceans and continents. Monthly weather review, 105(7), pp.915-929.
+!>
+!
+!-----------------------------------------------------------------------
+
+  subroutine post_atmospheric_forcing(block)!{{{
+
+    type (block_type), pointer :: block
+
+    type (MPAS_time_type) :: currentForcingTime
+
+    type (mpas_pool_type), pointer :: &
+         mesh, &
+         forcingPool, &
+         timeVaryingForcingPool, &
+         diagnosticsPool
+
+    real(kind=RKIND), dimension(:), pointer :: &
+         windSpeedU, &
+         windSpeedV, &
+         windSpeedMagnitude, &
+         windStressZonal, &
+         windStressMeridional, &
+         atmosPressure, &
+         atmosphericPressure
+
+    real(kind=RKIND) :: &
+         windStressCoefficient, &
+         rhoAir, &
+         ramp, &
+         currentTime
+
+    real(kind=RKIND), pointer:: &
+        config_forcing_ramp, &
+        daysSinceStartOfSim
+
+    integer, pointer :: &
+         nCells
+
+    integer :: &
+         iCell
+
+    call MPAS_pool_get_config(block % configs, "config_time_varying_atmospheric_forcing_ramp", config_forcing_ramp)
+
+    call MPAS_pool_get_subpool(block % structs, "mesh", mesh)
+    call MPAS_pool_get_subpool(block % structs, "forcing", forcingPool)
+    call MPAS_pool_get_subpool(block % structs, "timeVaryingForcing", timeVaryingForcingPool)
+    call MPAS_pool_get_subpool(block % structs, "diagnostics", diagnosticsPool)
+
+    call MPAS_pool_get_dimension(mesh, "nCells", nCells)
+
+    call MPAS_pool_get_array(diagnosticsPool, "daysSinceStartOfSim", daysSinceStartOfSim)
+    call MPAS_pool_get_array(timeVaryingForcingPool, "windSpeedU", windSpeedU)
+    call MPAS_pool_get_array(timeVaryingForcingPool, "windSpeedV", windSpeedV)
+    call MPAS_pool_get_array(timeVaryingForcingPool, "windSpeedMagnitude", windSpeedMagnitude)
+    call MPAS_pool_get_array(forcingPool, "windStressZonal", windStressZonal)
+    call MPAS_pool_get_array(forcingPool, "windStressMeridional", windStressMeridional)
+
+    call MPAS_pool_get_array(timeVaryingForcingPool, "atmosPressure", atmosPressure)
+    call MPAS_pool_get_array(forcingPool, "atmosphericPressure", atmosphericPressure)
+
+    rhoAir = 1.225_RKIND
+
+    ramp = tanh((2.0_RKIND*daysSinceStartOfSim)/config_forcing_ramp)
+
+    do iCell = 1, nCells
+
+       windSpeedU(iCell) = ramp*windSpeedU(iCell)
+       windSpeedV(iCell) = ramp*windSpeedV(iCell)
+       windSpeedMagnitude(iCell) = sqrt(windSpeedU(iCell)**2 + windSpeedV(iCell)**2)
+       windStressCoefficient = (0.75_RKIND + 0.067_RKIND * windSpeedMagnitude(iCell)) * 0.001_RKIND * rhoAir  ! Garratt 1977
+
+       windStressZonal(iCell) = windSpeedU(iCell) * windSpeedMagnitude(iCell) * windStressCoefficient
+       windStressMeridional(iCell) = windSpeedV(iCell) * windSpeedMagnitude(iCell) * windStressCoefficient
+
+       atmosphericPressure(iCell) = ((1.0_RKIND-ramp)*101325.0_RKIND + ramp*atmosPressure(iCell)) - 101325.0_RKIND
```

Comment:
We try to avoid 'magic numbers' for constants.  I'm adding a constant here:
```
src/core_ocean/shared/mpas_ocn_constants.F
+       atm_ref_pressure   = 101325.0_RKIND    ! (Pa)
```